scripts/build-toolchains.sh

```diff
@@ -8,15 +10,83 @@ unamestr=$(uname)
 RDIR=$(pwd)
 : ${CHIPYARD_DIR:=$(pwd)} #default value is the PWD unless overridden
 
-if [ $# -ne 0 ]; then
-  TOOLCHAIN=$1
-  if [ $1 == "riscv" ]; then
-    TOOLCHAIN="riscv-tools"
-  elif [ $1 == "hwacha" ]; then
-    TOOLCHAIN="esp-tools"
-  fi
-else
-  TOOLCHAIN="riscv-tools"
+function usage
+{
+    echo "usage: ./scripts/build-toolchains.sh [riscv] [hwacha] [ firesim | --firesim] [--submodules-only]"
```

Comment:
missing `ec2fast` in the usage?

Reply:
also do you say ... `firesim` at any point in the args

Reply:
It's going to be removed now.

Reply:
Forgot that from an earlier iteration. I pushed a fix

Reply:
do you ever use `--submodules-only`

Reply:
fixed